src/components/calcite-input/calcite-input.scss

```diff
@@ -4,23 +4,23 @@
   & input,
   & .calcite-input-prefix,
   & .calcite-input-suffix {
-    @apply text--2 p-2 h-8;
+    @apply text--2h py-1 px-2 h-6;
   }
   & textarea {
-    min-height: 32px;
+    min-height: theme("minHeight.6");
```

Comment:
s/minHeight/height/

Reply:
I'm not sure what this means

Reply:
Replace `minHeight` here w/`height` (tailwind is silently failing here and not including the min-height because `minHeight.6` does not exist).

Reply:
Thanks for clarifying